lib/stdlib.ngs

```diff
@@ -1132,7 +1132,7 @@ TEST [1,2,3,1,2].take(X<3) == [1,2]
 
 # Would look much better with an iterator of e.
 doc EXPERIMENTAL! Do not use!
-doc Filters out items that match the pattern at the begginning of e.
+doc Filters out items that match the pattern at the beginning of e.
```

Comment:
No problem but next time please make separate pull request for unrelated changes